.vscode/tasks.json

```diff
@@ -0,0 +1,10 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
Yep. Only for JSON config files, so that various options can be documented. The comments are stripped before the JSON is parsed by the editor.